js/genkit/src/tool.ts

```diff
@@ -16,6 +16,7 @@
 
 export {
   asTool,
+  dynamicTool,
```

Comment:
This should probably be exported from `"genkit"`, no? vs. a sub-export

Reply:
done.

Reply:
I mean, exported from `genkit` as well